src/playbooks/foreman-proxy/foreman-proxy.yaml

```
@@ -0,0 +1,17 @@
+---
+- name: Setup Foreman Proxy
```

Nit: needs to be 2 words here. See https://grammarist.com/spelling/set-up-vs-setup/ for more details.
```suggestion
- name: Set up Foreman Proxy
```